vocabularies/earth-science-data-category.ttl

```diff
@@ -155,6 +179,27 @@ esdc:biogeology a skos:Concept ;
     skos:scopeNote "includes, for example, the interactions of various bacteria that are responsible for the formation of some minerals such as pyrite, and can concentrate economically important metals such as tin and uranium."@en ;
     skos:prefLabel "Biogeology"@en .
 
+esdc:carbon-geostorage-geology a skos:Concept ; 
+    skos:altLabel "carbon geostorage"@en,
+        "geologic sequestration"@en,
+        "geological sequestration"@en,
+        "carbon geosequestration"@en,
+        "carbon geo-sequestration"@en,
+        "co2 sequestration geology"@en,
+        "co2 sequestration"@en,
+        "co2 geosequestration"@en,
+        "co2 geo-sequestration"@en,
+        "co2 geostorage geology"@en, 
+        "co2 geostorage"@en ; 
+# This list of altLabels can be potentially endless, but please feedback on the above; also,do we add carbon storage reservoir geology, and all the variations of this, as altLabel/s?#
```

Comment:
```suggestion
```
I think we leave reservoir geology as it is. Assessment of the reservoir as far as poro-perm etc should be independent of what the purpose of the reservoir is. Let's leave the altLabels as they are too for the moment. As you say they could potentially be endless so we should probably stop at this level and wait for the extras people may actually use and request to be added.